### ISSUE TITLE
[4.0] com_media check all [a11y]

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#34204]
The select all checkbox in the media manager is a link disguised as a checkbox. It should not be a link at all it should be a checkbox. 
A checkbox can be selected with a space a link can only be selected with an enter.
That is why we have checkboxes everywhere and not links disguised as checkbox

**Comment by Krshivam25** [comment]
@brianteeman @chmst As per my understanding. I found checkbox is controlled by space buttons not enter button. If I'm wrong please add some description or Screenshots. So I can work on this issue.

**Comment by brianteeman** [comment]
Yes a checkbox is only selected by a space. That was the point. Using a link is wrong as that is selected by the enter

**Comment by Krshivam25** [comment]
This means that elements that are presented as buttons, for example, must respond to both Enter and Spacebar key presses. In some browsers, this may require keypress detection with scripting.

**Comment by brianteeman** [comment]
>  In some browsers, this may require keypress detection with scripting.

No - just use a button - nothing to script then

**Comment by laoneo** [comment]
Please test #34597.
